commcare_connect/microplanning/migrations/0011_remove_workareainaccessibilityrequest_unique_xform_work_area_inaccessibility_and_more.py

```
@@ -0,0 +1,20 @@
+    ]
+
+    operations = [
+        migrations.RemoveConstraint(
+            model_name="workareainaccessibilityrequest",
+            name="unique_xform_work_area_inaccessibility",
+        ),
+        migrations.AddConstraint(
+            model_name="workareainaccessibilityrequest",
+            constraint=models.UniqueConstraint(fields=("work_area",), name="unique_work_area_inaccessibility"),
+        ),
+    ]
```

Ruff (0.15.14)

[warning] Mutable default value for class attribute

(RUF012)